onward/conf/routes

```diff
@@ -62,6 +62,7 @@ GET        /gallery/most-viewed                       controllers.MostViewedGall
 # Experimental
 GET        /cards/opengraph/*path.json                controllers.CardController.opengraph(path)
 GET        /tagged.json                               controllers.TaggedContentController.renderJson(tag: String)
+GET        /series/mf2/*path.json                     controllers.SeriesController.renderMf2SeriesStories(path)
```

Comment:
not sure if it's worth putting the mf2 at the end because it's only the format that's changing not the content?  e.g. /series/*path.mf2.json or something similar